frappe/core/doctype/user/user.py

```diff
@@ -44,6 +44,29 @@ def before_insert(self):
 	def after_insert(self):
 		self.set_default_roles()
 
+	def force_user_email_update(self):
+		for user_email in self.user_emails:
+			if not user_email.email_id:
+				user_email.email_id = frappe.db.get_value("Email Account", {"name": user_email.email_account},
+				                                          "email_id")
+
+	def user_emails_to_permissions(self):
+		if frappe.session.user == "Administrator" or "System Manager" in frappe.get_roles():
```

Comment:
in outlook people often have multiple email accounts added it's intended as that use
ie name@company.com and sales@company.com
multiple people can access to sales email account

maybe having the system handle the communications permission full would be best have a checkbox and remove from role permission? to force the update?